changelog/unreleased/kong/set_grpc_tls_seclevel.yml

```diff
@@ -0,0 +1,3 @@
+message: When ssl_cipher_suite is set to 'old', the security level of gRPC's TLS is also set to 0
```

Comment:
Actually, I can't understand what this message really means.

Reply:
Please let the message start with a verb, otherwise I'm having trouble understanding what this PR has done.

Reply:
fixed